tes-adapter/scripts/run_get_tes-adapter_jar.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+workdir=$(pwd);
+
+if [ -n "$2" ]; then
+    gitRepoUrl="-b $1 $2"
+else
+    gitRepoUrl=$1
+fi
+
+mkdir tmp;
+cd tmp;
+
+tmpdir=$(pwd);
+
+git clone $gitRepoUrl $tmpdir;
+
+./gradlew bootJar;
```

Comment:
in the automation script we do not need to see all output that is generated by tools, lets hide it and simply notify user that we start some operation and done it, or fail if something wrong

Reply:
fixed